tools/ja-rule/USBDeviceManager.cpp

```diff
@@ -180,7 +183,7 @@ void USBDeviceManager::SignalEvent(EventType event, JaRuleWidget* widget) {
     return;
   }
 
-  if (pthread_equal(m_start_thread_id, Thread::Self())) {
+  if (m_in_start && pthread_equal(m_start_thread_id, Thread::Self())) {
```

Comment:
http://stackoverflow.com/questions/6276939/is-there-an-invalid-pthread-t-id and http://www.linuxforums.org/forum/programming-scripting/123602-pthread_t-question.html

If you're happy with just "m_start_thread_id = pthread_t()" we can stick with that.